### DISCUSSION TITLE
Documentation for AWA high-speed protocol?

### DISCUSSION THREAD

**Discussion by sofakng** [awawa-dev/HyperHDR#171] [Q&A]
Is there any documentation or description for the AWA high-speed protocol?

I've read through the code (ie. HyperSerialESP32) but it would also be helpful to read any other documentation or description if available.

I don't think I have the programming skills required (ie. I'm mostly a desktop/web developer) but I'd like to try implementing this into [SignalRGB](https://www.whirlwindfx.com/pages/signalrgb) or even try maintaining a fork of WLED.

**Comment by awawa-dev** [comment]
No, there is not. Instead working code from HyperHDR (server) or HyperSerialEsp(client) can be use. WLED fork already exists (HyperSerialWLED)

**Reply by sofakng** [reply]
Thanks.  I saw the WLED fork but you mentioned not providing any further updates.  Is this still true or would you consider updating it again?  (maybe the next major release of WLED?)

**Reply by awawa-dev** [reply]
there are released HyperSerialWLED's  updates based up to WLED beta 0.13.0-b5.
There is a mess with WLED pinout remapping on ESP8266 and it only causes troubles when highrefresh rate is considered and it's poorly documented in WLED. HyperSerialWLED based 0.13 will be released at least but currently  don't see a point in next upgrades.
